api/users.rb

```diff
@@ -103,6 +104,10 @@ def _user_social_stats(user_id, params)
       content_selector[:created_at.lte] = end_date
     end
 
+    if user_ids
+        content_selector[:author_id.in] = user_ids
+    end
```

Comment:
content_selector is the data selection criteria, if you see line 133 its where it is used to fetch data from database